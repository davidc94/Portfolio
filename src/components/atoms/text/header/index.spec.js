import React from 'react';
import renderer from 'react-test-renderer';
import StyledHeader from './index';

describe('StyledHeader component', () => {
  it('render default StyledHeader as expected', () => {
    const render = renderer.create(
      <StyledHeader>I have nothing to offer</StyledHeader>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('render StyledHeader with a different font size', () => {
    const render = renderer.create(
      <StyledHeader size="large">EXTRA BIG</StyledHeader>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});