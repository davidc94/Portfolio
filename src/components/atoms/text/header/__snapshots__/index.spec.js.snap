// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHeader component render StyledHeader with a different font size 1`] = `
<h1
  className="sc-bdVaJa jPLkst"
  size="large"
>
  EXTRA BIG
</h1>
`;

exports[`StyledHeader component render default StyledHeader as expected 1`] = `
<h1
  className="sc-bdVaJa cEyAFy"
>
  I have nothing to offer
</h1>
`;
